Makefile

```diff
@@ -176,8 +180,9 @@ localnet-start: localnet-stop build-docker-localnode
 localnet-stop:
 	docker-compose down
 
-###########################################################
-### Remote full-nodes (sentry) using terraform and ansible
+###############################################################################
+###          Remote full-nodes (sentry) using terraform and ansible         ###
+###############################################################################
 
 # Server management
 sentry-start:
```

Comment:
this is unsupported imo. not sure if it's working now. should we consider removing?

Reply:
maybe we don't do it within this PR just to keep the scope narrow, but we can have a follow-up issue and discussion about what and IF we want to support these kind of commands.